kubernetes/cray-kafka-operator/Chart.yaml

```diff
@@ -1,6 +1,19 @@
-apiVersion: v1
-appVersion: 0.15.0
+apiVersion: v2
+version: 0.7.0
 description: An extension of the official kafka-operator helm chart
 name: cray-kafka-operator
-home: "cloud/cray-charts"
-version: 0.4.1
+keywords:
+  - kafka-operator
+home: https://github.com/Cray-HPE/cray-kafka-operator
+sources:
+  - https://github.com/strimzi/strimzi-kafka-operator
+maintainers:
+  - name: brantk-hpe
```

Comment:
Not sure why I'm the maintainer of this I have never contributed to it.

Reply:
Changed.